cli/flags.rs

```diff
@@ -1082,6 +1102,60 @@ Future runs of this module will trigger no downloads or compilation unless
     )
 }
 
+fn cover_subcommand<'a, 'b>() -> App<'a, 'b> {
+  SubCommand::with_name("cover")
```

Comment:
As discussed offline, should we rename to `coverage`?

Reply:
Rename the command and test prefixes to coverage.